packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/formDialog.tests.js

```diff
@@ -45,14 +46,24 @@ QUnit.module('FormDialog', moduleConfig, () => {
         assert.strictEqual(wrapperHasDialogClass, true, 'Wrapper with the FormDialog class');
     });
 
+    test('Dialog wrapper has dx-dropdowneditor-overlay class', function(assert) {
```

Comment:
add a ticket ID to demonstrate real scenario plz

Reply:
[4dd0459fcef9b023c0d6bba5df0c8936f8fbe512]